src/main/java/domain/CarGame.java

```diff
@@ -0,0 +1,56 @@
+package domain;
+
+import view.Result;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarGame {
+
+    // 자동차 객체 생성
+    public static List<Car> createCar(String[] carNames){
+        List<Car> cars = new ArrayList<>();
+        for (String name : carNames) {
+            cars.add(new Car(name));
+        }
+
+        return cars;
+    }
+
+    // 주어진 횟수 동안 실행
+    public static void moveCar(List<Car> cars, int n){
+        for (int i = 0; i < n; i++) {
+            for (Car car : cars) {
+                car.move();
+                Result.printCarDistance(car.getName(), car.getDistance());
+            }
+            System.out.println();
+        }
```

Comment:
> indent(인덴트, 들여쓰기) depth를 2를 넘지 않도록 구현한다. 1까지만 허용한다.

자동차 경주 미션의 요구사항으로, indent depth에 대한 제약이 있어요! 이 부분 말고도 다른 곳에서 이를 위반하는 코드가 존재하는데요, 어떻게 하면 indent depth를 줄일 수 있을까요?

이 코드를 예로 들어보자면, 여러 개의 Car를 지니는 Cars 라는 객체([일급 컬렉션](https://jojoldu.tistory.com/412))에 적절한 역할을 부여해서 indent depth를 줄여볼 수 있지 않을까요?

```suggestion
        Cars cars = new Cars();
        for (int i = 0; i < n; i++) {
            cars.move();
            Result.printCars(cars);
        }
```